frontend/micro-ui/web/src/Customisations/UICustomizations.js

```
@@ -0,0 +1,1018 @@
+            );
+          } else if (counter >= 1) {
+            return (
+              <PayAndDownloadButton
+                tenantId={tenantId}
+                certificateId={row?.id}
+              />
+            );
+          }
+          return <span>{t("ES_COMMON_NA")}</span>;
+        case "BND_DEATH_DATE":
+          const epoch = row?.dateofdeath;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)